test/jdk/java/beans/XMLEncoder/ReferenceToNonStaticField.java

```diff
@@ -26,7 +26,7 @@
 /**
  * @test
  * @bug 8060027
- * @run main/othervm -Djava.security.manager=allow ReferenceToNonStaticField
+ * @run main/othervm ReferenceToNonStaticField
```

Comment:
Other tests in the set removed `/othervm`. Is it left intentionally?

Reply:
@aivanov-jdk 
It was missed when -Djava.security.manager=allow was removed. 
Out of curiosity: does it affect the performance of CI testing if tests are run in `/othervm` mode when it is not needed?

Reply:
Right, it wasn't intentionally but missed when removing `-Djava.security.manager=allow`.

> Anyway, removing /othervm isn't strictly required because java/beans are run in othervm mode.

Thank you for clarifying. Since `/othervm` doesn't impact the beans tests I'll keep it as-is. Let me know if you still think it is good to remove, I can probably do a replace-all on beans/XMLEncoder/ folder.

Reply:
@honkar-jdk I've submitted [JDK-8343062](https://bugs.openjdk.org/browse/JDK-8343062) for additional cleanup.